js/views/SurveyDetailsPage.js

```diff
@@ -81,7 +111,6 @@ const SurveyDetailsPage = React.createClass ({
       survey.status = 'declined'
     })
     this.props.navigator.pop()
-    // this.props.navigator.push({path: 'surveylist', title: 'Surveys', sceneConfig: 'FloatFromLeft'})
   },
 
   showForms(tab) {
```

Comment:
Will do. Should we keep the code related to the form type menu? At this point it isn't used because we only have one tigger type and once a survey is selected, the user is directed to the first active form.